history/taskana-simplehistory-rest-spring/src/main/java/pro/taskana/simplehistory/rest/TaskHistoryEventController.java

```diff
@@ -152,7 +109,7 @@ public TaskHistoryEventController(
     }
 
     HistoryQuery query = simpleHistoryService.createHistoryQuery();
-    query = applySortingParams(query, params);
```

Comment:
According to clean code it is a good practice to return the changed value, even if you don't have to do it. What do you think?